src/actions/profileActions/profileActions.js

```diff
@@ -0,0 +1,113 @@
+import axios from 'axios';
+import {notify} from 'react-notify-toast';
+import requestLoadingAction from '../authActions/requestLoading';
+import { API_URLS, USERNAME_KEY } from '../../constants';
+import ACTION_TYPE from '../authActions/actionTypes';
+
+const NotificationSuccess = (message) => notify.show(message, 'success', 5000);
+
+const NotificationError = (message) => notify.show(message, 'error', 5000);
+
+export const fetchProfileSuccess = (userData) => ({
+
+	type: ACTION_TYPE.FETCH_PROFILE_SUCCESS,
+	payload: {
+		username: userData.profile.username,
+		firstName: userData.profile.first_name,
+		lastName: userData.profile.last_name,
+		bio: userData.profile.bio,
+		avatar: userData.profile.avatar
+	}
+
+});
+
+export const fetchProfileFailure = (error) => ({
+
+	type: ACTION_TYPE.FETCH_PROFILE_FAILURE,
+	payload: {
+		message: error
+	}
+
+});
+
+export const updateProfileSuccess = (successMessage) => ({
+
+	type: ACTION_TYPE.UPDATE_PROFILE_SUCCESS,
+	payload: {
+		message: successMessage
+	}
+
+});
+
+export const updateProfileFailure = (errorMessage) => ({
+
+	type: ACTION_TYPE.UPDATE_PROFILE_FAILURE,
+	payload: {
+		message: errorMessage
+	}
+
+});
+
+const handleErrors = (dispatch, props, message) => {
+
+	dispatch(requestLoadingAction(false));
+	NotificationError(message);
+	props.history.push('/logout');
+
+};
+
+const myHeader = {headers: {'Content-Type': 'application/json', 'Accept': 'application/json',
+	'Authorization': `Token ${ localStorage.getItem('auth_token')}`}};
```

Comment:
@collinewait and @DrKimpatrick, please use the global variables defined to access `username and access token` from local storage and we avoid hard-coding such. like **localStorage.getItem(USERNAME_KEY);** and **localStorage.getItem(AUTH_TOKEN);**

Reply:
It has been resolved.